这里存档,如何处理错误信息1的status是true,2是fasle

```
/// 测试报错接口--24.12-02-Apipost接口名称 错误测试接口  
class TestPort extends ImageBaseRequest {  
  @override  
  HttpMethod httpMethod() {  
    return HttpMethod.get;  
  }  
  
  @override  
  String path() {  
    return "/test_error";  
  }  
}
```

```
class TestRequest {  
  ///测试报错接口--24.12-02-Apipost接口名称 错误测试接口  
  static Future<TestError> testError(int pid) async {  
    TestPort request = TestPort();  
    request.addParams('id', pid);  
    var result = await OhNet.getInstance().fire(request);  
    TestError list = TestError.fromJson(result);  
    return list;  
  }  
}
```



```
TextButton(  
    onPressed: () async {  
      var result;  
      try {  
        result = await TestRequest.testError(2);  
        if (result.status!) {  
          print('测试错误接口结果正常');  
        } else {  
          print('测试错误接口结果不正常${result}');  
        }  
      } catch (e) {  
        print(e);  
        print('测试错误接口连接不上');  
      }  
    },  
    child: Text('测试错误接口')),
```
如果我的请求时错误的,会抛出一个错误,,这个时候会触发` print('测试错误接口结果不正常');` 


如果我的请求时1,会正确返回true,,这个时候会触发`print('测试错误接口结果正常');` 
```
[OH_NET] url:http://106.12.75.80:8091/test_error
[OH_NET] options:{}
[OH_NET] params:{"id":"1"}
[OH_NET] data:{}
I/flutter (13571): 测试错误接口结果正常
[OH_NET] result:{status: true, code: 200, message: 测试成功信息, data: null}
```
如果我的请求时2,会正确返回fasle,,这个时候会触发`print('测试错误接口结果正常');` 
```
[OH_NET] url:http://106.12.75.80:8091/test_error
[OH_NET] options:{}
[OH_NET] params:{"id":"2"}
[OH_NET] data:{}
I/flutter (13571): 测试错误接口结果不正常Instance of 'TestError'
[OH_NET] result:{status: false, code: 400, message: 测试错误信息, data: null}
```
如果我的请求完全不正确(接口错误时),会返回一个报错,,这个时候会触发`print('测试错误接口连接不上');` 
```
@override  
String path() {  
  return "/test_erro1r";  
}
```

所以可以基于这三种来进行解决报错和生成成功或者失败